ext/gd/gd.c

```diff
@@ -3819,7 +3818,7 @@ PHP_FUNCTION(imageaffinematrixget)
 	zval *tmp;
 	int res = GD_FALSE, i;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "l|z", &type, &options) == FAILURE)  {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "lz", &type, &options) == FAILURE)  {
```

Comment:
There is an `!options` check a few lines below, which can also be dropped.

Reply:
Thank you! I forgot about it :)